frontend/src/components/project/project-month-list/open-list/OpenedProjectMonthsList.tsx

```diff
@@ -26,15 +26,14 @@ const createFiltersSelector = () => createSelector(
   (filters) => {
     const safeFilters: FiltersModel = {
       ...filters,
-      openMonths: {},
     }
     return safeFilters;
   }, {
     // New in 4.1: Pass options through to the built-in `defaultMemoize` function
     memoizeOptions: {
       // TODO: need to update for new filters here! use deep-equal instead.
       // --> or better, the feature has the filter method there!
-      equalityCheck: (a: FiltersModel, b: FiltersModel) => a.freeText === b.freeText && a.unverifiedOnly === b.unverifiedOnly,
+      equalityCheck: (a: FiltersModel, b: FiltersModel) => a.freeText === b.freeText && a.unverifiedOnly === b.unverifiedOnly && a.sort === b.sort,
```

Comment:
Welke type is a.sort en b.sort hier?
Zijn dat strings? Of objecten? Of functions?